alacritty/src/input.rs

```diff
@@ -384,6 +384,11 @@ impl<'a, T: EventListener, A: ActionContext<T>> Processor<'a, T, A> {
         let cell_changed =
             point.line != self.ctx.mouse().line || point.col != self.ctx.mouse().column;
 
+        // Update mouse state and check for URL change.
+        let mouse_state = self.mouse_state();
+        self.update_url_state(&mouse_state);
+        self.ctx.window_mut().set_mouse_cursor(mouse_state.into());
```

Comment:
This is technically a slight performance degradation, since we now need to check for URLs and message bar every mouse motion and not only when the cursor moved by at least half a cell. Realistically though since URL parsing doesn't occur I don't think it should make a difference.

Reply:
Could you elaborate why changes to message bar were required and where it was broken on master? if it was broken, please, add a changelog entry for it.

Reply:
We don't check for cell size changes anymore because there are no cells where the message bar goes?

This has nothing to do with master being broken.